docs/enterprise/thales-install-guide.md

```diff
@@ -44,23 +51,29 @@ You must also perform the following steps:
    the SEE machine. **Important**: this hash is of a _development key_ and
    should not be trusted for production.
    
--  On the client host (RFS server), copy the signed firmware and userdata files to the custom-seemachines directory. 
-   Here, `a` and `b` have been chosen to simplify entering the file names on the HSM front panel. 
-   This command must be run on each module.
+-  On the client host (RFS server), copy the signed firmware and userdata files to the custom-seemachines directory. These commands must be run on each module.
+-  Here, `a` and `b` have been chosen to simplify entering the file names on the HSM front panel. These names can be changed if desired.
    - `cp xprvseemodule.sar /opt/nfast/custom-seemachines/a`
    - `cp userdata.sar /opt/nfast/custom-seemachines/b`
 
 -  On the HSM, use the front panel to load the machine image and userdata file. 
-   This must be done for each HSM.
+   This must be done for each HSM. Note: there is no need to reboot the modules.
    - Choose `CodeSafe` on the front panel 
-   - Enter`a` for the machine image 
-   - Enter`b` for the userdata file
-   - Choose`SEElib` for the world type
-   - Enter`chainenclave` for the name
+   - Enter `a` (or name of choice) for the machine image 
+   - Enter `b` (or name of choice) for the userdata file
+   - Choose `SEElib` for the world type
+   - Enter `chainenclave` for the name
+
+-  On the client host, check that the SEE world is running on each module using stattree.
+   - `stattree PerModule 1 ModuleEnvStats`
```

Comment:
Maybe also say something like "the MemAllocUser value in the output should be nonzero".